packages/broker/test/unit/ConfigWizard.test.ts

```diff
@@ -8,6 +8,19 @@ const mockPromptMethod = (wizard: ConfigWizard, mockedPromptResult: any) => {
 }
 describe('ConfigWizard', () => {
     let wizard: ConfigWizard
+    const tmpDataDir = '/tmp/streamr-broker-test/'
```

Comment:
Would be a bit more flexible to rely on ´os.tmpdir()` instead to fetch temp directory (which could be different depending on operating system)

Reply:
didn't know about `os.tmpdir()`, right what was needed!